manifests/module.pp

```diff
@@ -44,7 +47,7 @@
     require ::selinux::refpolicy_package
   }
 
-  if ($builder == 'simple' and $source_if != undef) {
+  if ($builder == 'simple' and ($source_if != undef or $content_if != undef)) {
```

Comment:
Not exactly (think empty string). Anyway, I just copied original code style. Do you think it must be changed?